docs/overview.adoc

```diff
@@ -607,34 +607,6 @@ CALL apoc.periodic.rock_n_roll('match (p:Person) return id(p) as id_p', 'MATCH (
 
 // end::periodic[]
 
-== Graph Refactoring
```

Comment:
We should still have the overview list here. Just no details. You can also include:: a tagged region of `graph-refactor.adoc`